CHANGELOG.md

```diff
@@ -11,7 +11,8 @@
 * Introducing `AbstractVar` to abstract value access: store, load, and stack type. ([#584](https://github.com/algorand/pyteal/pull/584))
   * NOTE: a backwards incompatable change was imposed in this PR: previous ABI value's public member `stored_value` with type `ScratchVar`, is now changed to protected member `_stored_value` with type `AbstractVar`.
 * Starting with program version 9, when `scratch_slots` flag isn't provided to `OptimizeOptions`, default to optimizing. For versions 8 and earlier the default is and remains to _not_ optimize. ([#613](https://github.com/algorand/pyteal/pull/613))
-* Replaced the usage of `typing.NamedTuple` with `dataclass` for `class OpType` in the **ir** package in order to avoid [a regression coming in Python 3.11.1](https://github.com/python/cpython/issues/100098) ([#615](https://github.com/algorand/pyteal/pull/615)).
+* Replaced the usage of `typing.NamedTuple` with `dataclass` for `class OpType` in the **ir** package in order to avoid [a regression coming in Python 3.11.1](https://github.com/python/cpython/issues/100098). ([#615](https://github.com/algorand/pyteal/pull/615))
+* Upgrade to the latest mypy. ([#618](https://github.com/algorand/pyteal/pull/618))
```

Comment:
Optional - If we're taking the effort to call out the dependency upgrade, I think the version should be qualified.

```suggestion
* Upgrade mypy to v0.991. ([#618](https://github.com/algorand/pyteal/pull/618))
```